recipes/python3/ModulesSetup

```diff
@@ -2,9 +2,11 @@
 # Static compilation instructions for all binary modules.
 #####################################################################
 
+*static*
```

Comment:
What is that?

Reply:
From *python* `Setup.dist`:

```# Each line in this file describes one or more optional modules.
# Modules configured here will not be compiled by the setup.py script,
# so the file can be used to override setup.py's behavior.
# Tag lines containing just the word "*static*", "*shared*" or "*disabled*"
# (without the quotes but with the stars) are used to tag the following module
# descriptions. Tag lines may alternate throughout this file.  Modules are
# built statically when they are preceded by a "*static*" tag line or when
# there is no tag line between the start of the file and the module
# description.  Modules are built as a shared library when they are preceded by
# a "*shared*" tag line.  Modules are not built at all, not by the Makefile,
# nor by the setup.py script, when they are preceded by a "*disabled*" tag
# line.
```

Basically, considering that is something by default, We can safely remove this line ( Already tried and it works, I just missed it over there during my tests)